assets/blocks/course-outline/style.scss

```diff
@@ -54,68 +41,161 @@ $dark-gray: #1a1d20;
 }
 
 .wp-block-sensei-lms-course-outline-module {
-	border: solid 1px $dark-gray;
-	margin: 1.5em 0;
+	$block: '.wp-block-sensei-lms-course-outline-module';
+
+	&.is-style-default, &.is-style-minimal {
+
+		#{$block}__progress-indicator {
+			border-radius: 999px;
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			margin-right: 20px;
+			margin-left: 20px;
+
+			&__text {
+				font-weight: normal;
+				margin: 2px 15px;
+				font-size: 14px;
+				white-space: nowrap;
+				text-transform: uppercase;
+			}
+
+			&.completed {
+				background-color: #1FD25A;
+				color: #fff;
+			}
+		}
 
-	&__name {
-		padding: 15px 15px 15px 25px;
-		background-color: $dark-gray;
-		font-size: 24px;
-		font-weight: bold;
-		color: #FFF;
-		display: flex;
-		align-items: center;
+		#{$block}__name {
+			padding: 1rem;
+			font-weight: bold;
+			display: flex;
+			align-items: center;
+		}
+
+		h2 {
+			font-family: inherit;
+			font-size: inherit;
+			color: inherit;
+			line-height: inherit;
+			margin: 0;
+			flex: 1;
+		}
+
+		#{$block}__description {
+			padding: 1rem;
+		}
+
+		#{$block}__lessons-title {
+			padding: 5px 1rem;
+			font-size: 11px;
+			font-weight: 600;
+			color: inherit;
+			text-transform: uppercase;
+		}
 	}
 
-	&__description {
-		padding: 25px;
-		color: $dark-gray;
-		font-size: 14px;
+	&.is-style-default {
+
+		#{$block}__lessons-title {
+			background-color: #EEE;
+		}
+		#{$block}__name {
+			&:not(.has-background-color) {
+				background-color: $dark-gray;
+			}
+			&:not(.has-color) {
+				color: #FFF;
+			}
+		}
+
+		#{$block}__lessons-title {
+			background-color: #EEE;
+		}
+		#{$block}__progress-indicator {
+			background-color: #FFF;
+			color: $dark-gray;
+		}
 	}
 
-	&__lessons-title {
-		padding: 5px 25px;
-		background-color: #EEE;
-		font-size: 11px;
-		font-weight: 600;
-		color: $dark-gray;
-		text-transform: uppercase;
+	&.is-style-minimal {
+		#{$block}__progress-indicator:not(.completed) {
+			border: 1px solid currentColor;
+		}
+
+	}
+
+	&__name__minimal-border {
+		display: block;
+		border: none;
+		height: 4px;
+		.wp-block-sensei-lms-course-outline.is-style-default & {
+			margin-left: 4px;
+			margin-right: 4px;
+		}
+		background-color: $dark-gray;
 	}
 }
 
 .wp-block-sensei-lms-course-outline-lesson {
 	display: flex;
 	align-items: center;
-	padding: 20px 25px;
+	margin: 1px 0;
+	position: relative;
+	font-size: 1rem;
```

Comment:
I was trying to get away from `px` for things that would better be dictated by the theme, but good catch, this should be `em` at least. 

Stepping back, maybe it's better after all to not pick up the theme heading styles for `h2`, `h3`, since we have some extra styling around them. I'll try a solution where we have the theme defaults for the *Unstyled* variant, and add some stricter sizing to the other styles.

I'm a bit torn on whether we should even use `h3` for the lesson blocks, since they are not really headings in this context.

Reply:
It's likely impacting other parts using `rem` too.

Reply:
> I'm a bit torn on whether we should even use h3 for the lesson blocks, since they are not really headings in this context.

I agree with you! It's not title in this context. It doesn't need to be an `h3`.

Reply:
But in the case of the `Lessons` title, I think it should be a heading.

Reply:
Adjusted the headings mostly in dc1bb75
Also looks like the more generic overrides could be removed now 72f42bc

Reply:
I checked there are some more `rem` in spacings which can be impacted differently depending on the theme. I think we should replace all of them. Or did you keep them on purpose for some reason?

Reply:
I was thinking it's fine for the theme to provide, but probably not with root font sizes being set to weird values. Also not common in built-in component styles, so changed these to `px` (and put it in a variable where they should be the same for proper alignment)